solidity/truffle.js

```diff
@@ -1,26 +1,62 @@
-/*
- * NB: since truffle-hdwallet-provider 0.0.5 you must wrap HDWallet providers in a
- * function when declaring them. Failure to do so will cause commands to hang. ex:
- * ```
- * mainnet: {
- *     provider: function() {
- *       return new HDWalletProvider(mnemonic, 'https://mainnet.infura.io/<infura-key>')
- *     },
- *     network_id: '1',
- *     gas: 4500000,
- *     gasPrice: 10000000000,
- *   },
- */
+const HDWalletProvider = require('truffle-hdwallet-provider')
 
 require('babel-register')
 require('babel-polyfill')
 
 module.exports = {
   networks: {
     development: {
+      host: '127.0.0.1',
+      port: 7545,
+      network_id: '*', // Match any network id,
+      gas: 6000000,
+      from: '0x94ad4774dc2da220088dc7431830a5ad77e4dcc4'
+    },
+    parity: {
       host: '127.0.0.1',
       port: 8545,
-      network_id: '*' // Match any network id,
+      network_id: '*',
+      gas: 6000000,
+      from: '0x94ad4774dc2da220088dc7431830a5ad77e4dcc4'
+    },
+    ropsten: {
```

Comment:
I need to fix up the launching of kovan and ropsten. i will get on it tomorrow. however if you need to merge this PR you can just delete or comment out this section, and ill make a new PR off the new master